backend/inspirehep/records/receivers.py

```diff
@@ -44,3 +45,8 @@ def index_after_commit(sender, changes):
                     author_disambiguation(model_instance)
                 if "new_record" in model_instance.json:
                     redirect_references_to_merged_record.delay(str(model_instance.id))
+                if (
+                    "documents" in model_instance.json
+                    and current_app.config["FEATURE_FLAG_ENABLE_FULLTEXT"]
+                ):
+                    index_fulltext.delay(str(model_instance.id))
```

Comment:
shouldn't we only run it for literature?

Reply:
I did a check in a task for the record type but I can do it here (also don't think that we have `documents` in any other record)

Reply:
we will have in the `data` collection is coming on Q3